packages/webview_flutter/webview_flutter_web/example/integration_test/legacy/webview_flutter_test.dart

```diff
@@ -4,22 +4,34 @@
 
 import 'dart:async';
 import 'dart:html' as html;
+import 'dart:io';
 
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 import 'package:webview_flutter_web_example/legacy/web_view.dart';
 
-void main() {
+void main() async {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
-  // URLs to navigate to in tests. These need to be URLs that we are confident will
-  // always be accessible, and won't do redirection. (E.g., just
-  // 'https://www.google.com/' will sometimes redirect traffic that looks
-  // like it's coming from a bot, which is true of these tests).
-  const String primaryUrl = 'https://flutter.dev/';
-  const String secondaryUrl = 'https://www.google.com/robots.txt';
+  const String primaryPage = 'first.txt';
+  const String secondaryPage = 'second.txt';
+  final HttpServer server = await HttpServer.bind(InternetAddress.anyIPv4, 0);
```

Comment:
Can this be:

```suggestion
  final HttpServer server = await HttpServer.bind(InternetAddress.loopbackIPv4, 0);
```

So you can connect only from `http://localhost:${server.port}`?

Reply:
I think the other version just opens a random port to the world that could potentially be connected to from the outside world? (I don't know the security ramifications, but if I do that internally at work I get an automated email flagging it :P)